shared/src/test/java/uk/gov/di/authentication/shared/services/AuthorizationServiceTest.java

```diff
@@ -444,6 +447,28 @@ void shouldGetPersistentCookieIdFromExistingCookie() {
         assertEquals(persistentSessionId, "some-persistent-id");
     }
 
+    @ParameterizedTest
```

Comment:
Could simplify these into a single test and make the `MethodSource` pass the expected value too.

Reply:
Good suggestion. This has now been updated.